src/server/scripts/Northrend/FrozenHalls/ForgeOfSouls/boss_bronjahm.cpp

```diff
@@ -21,13 +21,15 @@
 #include "SpellAuraEffects.h"
 #include "forge_of_souls.h"
 
-enum Yells
+#define DATA_SOUL_POWER    1
+
+enum Events
 {
-    SAY_AGGRO           = 0,
-    SAY_SLAY            = 1,
-    SAY_DEATH           = 2,
-    SAY_SOUL_STORM      = 3,
-    SAY_CORRUPT_SOUL    = 4,
+    EVENT_MAGIC_BANE      = 1,
+    EVENT_SHADOW_BOLT     = 2,
+    EVENT_CORRUPT_SOUL    = 3,
+    EVENT_SOULSTORM       = 4,
+    EVENT_FEAR            = 5
```

Comment:
[**MitchesD**](https://github.com/MitchesD) added a note [17 hours ago](https://github.com/TrinityCore/TrinityCore/pull/14036#discussion_r24184413 "Feb. 5. 2015, 19:12 CET")
yeah it seems to be wrong too